stories/internal/DualAxesChartConfig.tsx

```diff
@@ -90,7 +93,31 @@ storiesOf('Internal/DualAxesMinMaxConfig', module)
             </div>
         );
     })
-    .add('Dataset with 0+ data on both axes, min >= max', () => {
+    .add('Dataset with 0+ data on both axes, min > max on left axis', () => {
```

Comment:
I see that there are many removed screenshots from stories (14) and only a few added (4). Could you please check and explain why these are no longer needed? Thanks :-)

Reply:
:+1: